cmd/kubeadm/app/constants/constants.go

```diff
@@ -178,6 +178,9 @@ const (
 	// TLSBootstrapTimeout specifies how long kubeadm should wait for the kubelet to perform the TLS Bootstrap
 	TLSBootstrapTimeout = 2 * time.Minute
 
+	// DefaultClusterName defines the default cluster name
+	DefaultClusterName = "kubernetes"
```

Comment:
this change makes sense to me.
i.e. removing the constant from the versions / defaults.go files.